src/viam/sdk/common/utils.hpp

```diff
@@ -23,6 +23,8 @@ const std::string kService = "service";
 const std::string kRDK = "rdk";
 const std::string kBuiltin = "builtin";
 
+using time_pt = std::chrono::time_point<long long, std::chrono::nanoseconds>;
```

Comment:
Interesting. I'm trying to recall how we ended up with `long long` but I'm not finding it. I'll look into changing this to be a `clock` type and confirm that everything still runs as expected.

As for the typedef, can you elaborate a bit? I agree it doesn't save all that much (just some extra typing here or there) but are there costs or other reasons not to typedef that I'm not aware of?

Reply:
My thoughts on the typedef are individually minor but sort of add up to not wanting to do it:
- Every typedef has a cost when you need to figure out what it is.
- If we are standardizing on `std::chrono::time_point<X, Y>` as our standard time point type, I'd rather we just typedef it to`time_point` rather than `time_pt`.
- There is supposed to be a clock in there, and I think sorting that out should take precedence, since it may alter how the type gets used and undo some of the advantages of the typedef.

Reply:
cool makes sense! I went with a typedef in `utils.cpp` just to avoid having to type out the full type every time, but in headers and call sites reverted to the actual full type.

Also, tests seem to be passing fine when replacing `long long` with `steady_clock` so I'm going ahead and making that change.